documentation/assemblies/assembly-configuring-kafka-listeners.adoc

```diff
@@ -18,7 +18,7 @@ The following types of listener are supported:
 
 .{oauth}
 If you are using {oauth} token-based authentication, you can configure the listeners to connect to your authorization server.
-For more information, see xref:assembly-oauth-str[Using {oauth} token-based authentication].
+For more information, see xref:assembly-oauth-authentication_str[Using {oauth} token-based authentication].
```

Comment:
Underscore or hyphen?

Reply:
We're meant to be using underscore before {context} now. I've started with new or updated files.